config/repositories.yaml

```diff
@@ -33,14 +33,14 @@ repositories:
     master: https://ghcr.io/hewlettpackard/lustre-csi-driver
     useRemoteK: true
     remoteReference:
-      build: v0.0.8
+      build: v0.0.9
       url: https://github.com/HewlettPackard/lustre-csi-driver.git/deploy/kubernetes/%s/?ref=%s
   - name: lustre-fs-operator
     development: https://ghcr.io/nearnodeflash/lustre-fs-operator
     master: https://ghcr.io/nearnodeflash/lustre-fs-operator
     useRemoteK: true
     remoteReference:
-      build: v0.0.8
+      build: v0.0.10
```

Comment:
We skipped 9?

Reply:
Isn't lustre-fs-operator where the failed build issue occurred that caused an adjustment to the tag action?

Reply:
No, there wasn't a failure.  Instead, I noticed that the `create_release` job would run before the `build` job had completed, so that's when I began the effort to move `create_release` so it could depend on `build`.